types/warmup.d.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License.
+
+import { FunctionOptions, FunctionResult, FunctionTrigger, RetryOptions } from './index';
+import { InvocationContext } from './InvocationContext';
+
+export type WarmupHandler = (warmupContext: object, context: InvocationContext) => FunctionResult;
+
+export interface WarmupFunctionOptions extends WarmupTriggerOptions, Partial<FunctionOptions> {
+    handler: WarmupHandler;
+
+    trigger?: WarmupTrigger;
+
+    /**
+     * An optional retry policy to rerun a failed execution until either successful completion occurs or the maximum number of retries is reached.
+     * Learn more [here](https://learn.microsoft.com/azure/azure-functions/functions-bindings-error-pages)
+     */
+    retry?: RetryOptions;
```

Comment:
Retry isn't officially supported for all trigger types. I don't see warmup listed in the docs, so let's remove this property

https://learn.microsoft.com/en-us/azure/azure-functions/functions-bindings-error-pages?tabs=fixed-delay%2Cisolated-process&pivots=programming-language-javascript

Reply:
Yeah, I had removed it but then I flip-flopped. Removed it for good now.